spec/support/specialist_publisher_navigation_helpers.rb

```diff
@@ -7,12 +7,26 @@ def visit_aaib_create
     visit("#{Plek.new.find('specialist-publisher')}/aaib-reports/new")
   end
 
-  def set_aaib_occurence_date(date)
+  def visit_esi_index
+    visit("#{Plek.new.find('specialist-publisher')}/esi-funds")
```

Comment:
it would be worth extracting this repeated Plek call into a separate method. Also it should just be `Plek.find`, not `Plek.new.find`.